env.example

```diff
@@ -20,3 +20,5 @@ INTERNAL_API_BASE_URL=http://localhost:9696
 INTERNAL_SYNC_API_BASE_URL=http://localhost:9697
 NEXT_PUBLIC_APP_ENVIRONMENT="development"
 DEFAULT_SYNC_DAYS=31
+BUILD_DATE=2024-06-05T10:21:34Z
+MERGE_COMMIT_SHA=5f9ff895ad1d7805edcb22bfe2fcc6129e33bd8c
```

Comment:
Are we going to use these asa dummy values?

Reply:
yup, its an example value